specs/processes.table

```diff
@@ -34,6 +34,10 @@ schema([
 extended_schema(WINDOWS, [
     Column("is_elevated_token", INTEGER, "Process uses elevated token yes=1, no=0"),
 ])
+
+extended_schema(DARWIN, [
+    Column("unique_pid", BIGINT, "A 64bit pid from the XNU kernel that is never reused"),
```

Comment:
In sqlite `BIGINT` is just an alias to `INTEGER`. We have been randomly mixing them, but should try to make this more consistent using only `INTEGER`.

Reply:
Please document the sentinel value `-1`.

Reply:
When I changed to BIGINT it broke on Windows and was unable to fit some numbers into INTEGER so it is different in someway